test/board_test.rb

```diff
@@ -19,4 +19,12 @@ def test_it_has_cells
     assert 16, @board.cells.length
     assert Cell, @board.cells["A1"]
   end
+
+  def test_it_can_validate_coordinates
```

Comment:
*Note: maybe revisit later for user board size.

Reply:
I agree it's very much hard coded right now.